fjord/l10n/utils.py

```diff
@@ -0,0 +1,9 @@
+import re
+
+
+_COLLAPSE_RE = re.compile(r'\s+', re.UNICODE)
+
+
+def collapse_whitespace(message):
+    """Collapses consecutive whitespace into a single space"""
+    return _COLLAPSE_RE.sub(' ', message).strip()
```

Comment:
fwiw, "my version" is what the jinja2-django-tags project does and where I saw it and got interested about the differences.
https://github.com/MoritzS/jinja2-django-tags/blob/master/jdj_tags/extensions.py#L188